lib/domain/dtos/filters/LhcFillsFilterDto.js

```diff
@@ -12,10 +12,15 @@
  */
 const Joi = require('joi');
 const { validateRange } = require('../../../utilities/rangeUtils');
+const { validateTime } = require('../../../utilities/validateTime');
 
 exports.LhcFillsFilterDto = Joi.object({
     hasStableBeams: Joi.boolean(),
     fillNumbers: Joi.string().trim().custom(validateRange).messages({
         'any.invalid': '{{#message}}',
     }),
+    beamDuration: Joi.string().trim().min(8).max(8).custom(validateTime).messages({
```

Comment:
Could add here a pattern to match the timestamp format to clean up validate function.